utils/sample_qc.py

```diff
@@ -364,82 +364,139 @@ def infer_sex(
 
 
 def get_sex_expr(
+    ht: hl.Table,
     chr_x_ploidy: hl.expr.NumericExpression,
     chr_y_ploidy: hl.expr.NumericExpression,
     f_stat: hl.expr.NumericExpression,
-    xx_x_ploidy_cutoffs: Tuple[float, float] = (1.4, 2.25),
-    xy_x_ploidy_cutoffs: Tuple[float, float] = (0.5, 1.4),
-    xx_y_ploidy_cutoff: float = 0.1,
-    xy_y_ploidy_cutoffs: Tuple[float, float] = (0.15, 1.2),
-    yy_y_ploidy_cutoff: float = 1.3,
-    xxx_x_ploidy_cutoff:float = 2.5,
-    f_stat_female_cutoff: float = -0.2,
-    f_stat_male_cutoff: float = 0.2
+    f_stat_XX_cutoff: float = -0.2,
+    f_stat_XY_cutoff: float = 0.2,
+    lower_SD_cutoff: int = 5,
+    upper_SD_cutoff: int = 6,
+    Y_lower_bound: int = 2,
```

Comment:
I misinterpreted your comment earlier for the Y lower bound being 2*SDs (which ended up being a reasonable value for this dataset). I've changed the Y_lower_bound to 7

Reply:
Actually, scratch that. I changed the code per your suggestion below ("Add argue that here, it would make more sense to get the X and Y cutoffs for the different ploidies in this function rather than the cutoffs for males vs females") and redefined the ploidy cutoffs. The lower bound of Y coverage is now the upper bound of Y coverage in samples categorized roughly as "XX" by f-stat